src/utils/dom/appendNode.js

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default function appendNode(parent, node) {
+  return parent.appendChild(node);
+}
```

Comment:
I think this relates to my other comment talking about what should/should not be a utility and the slippery slope.

Reply:
I just wanted to be consistent with the other utility `createNode`, name wise, hence this almost trivial utility. Also while I don't have any checks here, I was thinking of adding `null` checks to make it more robust.